code/game/gamemodes/revolution/revolution.dm

```diff
@@ -5,6 +5,7 @@
 	extended_round_description = "Revolutionaries - Remove the heads of staff from power. Convert other crewmembers to your cause using the 'Convert Bourgeoise' verb. Protect your leaders."
 	required_players = 4
 	required_enemies = 3
+	ert_disabled = 1

```

Comment:
The goal of the revolution gamemode is to remove the heads of staff from power.
To prevent a ERT call, remove (a sufficient number) from power or convert them.